data/src/main/java/io/quarkus/qe/data/RepositoryEntity.java

```diff
@@ -32,4 +32,7 @@
     public Set<LabelEntity> labels;
     @OneToMany(mappedBy = "repository", cascade = CascadeType.ALL, orphanRemoval = true, fetch = FetchType.LAZY)
     public List<LogEntity> logs;
+    @ManyToOne(cascade = CascadeType.ALL, fetch = FetchType.EAGER)
```

Comment:
Cascade ALL is correct? We don't want to delete the quarkus version if the repository is deleted.